frontend/src/common/styles/button.css

```diff
@@ -1,7 +1,17 @@
 .button {
+    background-color: #657FB4;
     border-radius: 0.5rem;
     font-weight: bold;
     margin: auto;
     width: 12rem;
     height: 3rem;
+}
+
+.cancel-button {
+    background-color: #F3F3F3;
+    color: #474747;
+}
```

Comment:
Check this ticket https://www.notion.so/Sync-updated-design-6019f820711e4752ab2d237735df7db8
There's already such task to update colors. Moreover, you update it manually, but there's an option to make it unified over material ui.
If Kostya makes it, you need to rollback it or you will get conflicts

Reply:
Do i need to delete this style? Because when Kostya do this, i anyway delete it

Reply:
You can delete now